src/Compiler/Facilities/DiagnosticsLogger.fs

```diff
@@ -132,21 +132,21 @@ let ErrorWithSuggestions ((n, message), m, id, suggestions) =
 let ErrorEnabledWithLanguageFeature ((n, message), m, enabledByLangFeature) =
     DiagnosticEnabledWithLanguageFeature (n, message, m, enabledByLangFeature)
 
-let inline protectAssemblyExploration dflt f =
+let inline protectAssemblyExploration dflt ([<InlineIfLambda>] f) =
```

Comment:
Plus this cheeky addition.

Reply:
Should probably go into different pull request